paper/src/main/java/com/denizenscript/denizen/paper/events/TargetHitScriptEvent.java

```diff
@@ -0,0 +1,98 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import io.papermc.paper.event.block.TargetHitEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+
+public class TargetHitScriptEvent extends BukkitScriptEvent implements Listener {
```

Comment:
classname should match the Denizen name not the paper name - ie `TargetBlockHitScriptEvent`